assets/create-template/templates/default/asyncapi.yaml

```diff
@@ -0,0 +1,33 @@
+asyncapi: 2.6.0
```

Comment:
@AayushSaini101 I suggest to target version 3 instead, as we want people to adopt it.